readable.js

```diff
@@ -1,6 +1,9 @@
-var Stream = require('stream'); // hack to fix a circular dependency issue when used with browserify
+(function (){
+  try {
+    exports.Stream = require('st' + 'ream'); // hack to fix a circular dependency issue when used with browserify
```

Comment:
OK now i get it, thanks for the explaination :smile:. We should get this merged, because we really need to get this module up-to-date with core.